### DISCUSSION TITLE
Allow advdupe2 to remember if the prop has shadows or not.

### DISCUSSION THREAD

**Discussion by DanishGuy47** [wiremod/advdupe2#473] [Suggestions]
This issue primarily affects DarkRP and similar game modes, where pasted dupes are ruined by map shadows. Fortunately, the "Prop Shadow Remover" addon fixes this issue. You can find it here: https://steamcommunity.com/sharedfiles/filedetails/?id=1274272415&searchtext=shadow+remover. However, when you save a dupe with the shadows removed, these changes aren't retained, requiring you to remove shadows each time you paste a dupe. It would be fantastic if advdupe2 could remember whether the shadows were removed or not.

**Comment by DanishGuy47** [comment]
There is also another shadow remover addon: https://steamcommunity.com/sharedfiles/filedetails/?id=2152483083&searchtext=shadow+remover

**Comment by thegrb93** [comment]
Addons are responsible for adding their own duplicator support using duplicator.RegisterEntityModifier

**Reply by DanishGuy47** [reply]
Thank you, I will proceed with my journey lol.
